remote/advancedcontainers/engine-options.md

```diff
@@ -0,0 +1,63 @@
+---
+Order: 16
+Area: advancedcontainers
+TOCTitle: Container platforms
+PageTitle: Remote - Containers engine options
+ContentId:
+MetaDescription: Remote - Containers engine options
+DateApproved: 2/4/2022
+---
+
+# Other Container CLIs and Engines
+
+Remote - Containers interacts with CLIs; it makes no assumptions about how a container engine works and does not interact with container engines or daemon directly. You may use other Docker compliant CLIs with Remote - Containers, several of which are described below. Please note they are not officially supported.
+
+## Podman
+
+[Podman](https://podman.io/) 1.9+ is mostly compatible with Docker's CLI commands and therefore generally does work if you update the `Docker Path` setting (which may be accessed through **Remote > Containers: Docker Path**) to `podman` on Linux.
+
+![Docker Path setting](images/platform-options/docker-path-setting.png)
+
+However, certain tricks like [Docker-from-Docker do not work](https://github.com/containers/libpod/issues/4056#issuecomment-535511841) due to limitations in Podman. This affects the **Remote-Containers: Try a Development Container Sample...** and **[Remote- Containers: Clone Repository in Container Volume...](/docs/remote/containers.md#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume)** commands.
+
+Docker Compose is also not supported by Podman.
+
+## Docker Contexts
+
+[Docker Contexts](https://docs.docker.com/engine/context/working-with-contexts/) allow you to interact with different hosts - you can set up contexts for each host and switch between them.
+
+You create new contexts with `docker context create`. The current context can be changed using `docker context use <context>`.
+
+The [Docker extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) comes with `docker.host` (same as `DOCKER_HOST` env variable) and `docker.context` (same as `DOCKER_CONTEXT` env variable) user settings that are also honored by the Remote-Containers extension.
+
+> **Note**: The above settings are only visible when the Docker extension is installed. Without the Docker extension, Remote-Containers will use the current context.
+
+### Colima
+
+[Colima](https://github.com/abiosoft/colima) provides container runtimes on macOS and Linux. It's recommended to use Colima v0.2.2 or later so that VS Code can properly see containers running through Colima.
+
+Update settings in the Docker extension to use Colima:
+* `"docker.host": "unix:///Users/<username>/.colima/docker.sock",`
```

Comment:
It might be better to use Remote-SSH to connect to the Colima VM and clone repositories there to avoid getting the workspace folder on the Mac filesystem mounted readonly (the way I have seen it work by default on my machine). To do that I've added an entry to the `~/.ssh/config`, the problem with that is that the port changes after restarting the VM:
```
Host colima
    HostName localhost
    Port 56384
    IdentityFile ~/.lima/_config/user
    NoHostAuthenticationForLocalhost yes
```

@Chuxel might have additional tips.

Reply:
I updated the Colima section to fall under the SSH header and recommend using Remote-SSH with the context you shared. Also added the ssh config.

@chrmarti @Chuxel please let me know if you have any other feedback or tips!

Reply:
Updated based on the below, which ultimately removed the ssh information.